.github/workflows/build.yml

```diff
@@ -0,0 +1,57 @@
+name: Java CI with Gradle
+
+on:
+  push:
+    branches: [ "kotlin_multiplatform" ]
+  pull_request:
+    branches: [ "kotlin_multiplatform" ]
+
+permissions:
+  contents: read
+
+jobs:
+
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: gradle/wrapper-validation-action@v1
+      
+  test:
+    runs-on: ubuntu-latest
+    needs: validate
+    strategy:
+      fail-fast: false
+      matrix:
+        version: [11, 17]
```

Comment:
if you check the docs for actions/setup-java@v3 you can use matrix to test both standard jdk and kotlin together

Reply:
sounds good, can use this in the future for testing new versions of Kotlin, but for today, building against the latest only (though the outputs should be backward compatible on older versions for usage)